edgeless_bal/Cargo.toml

```diff
@@ -31,4 +31,5 @@ hyper = { version = "1.0.0-rc.4", features = ["full"] }
 hyper-util = { git = "https://github.com/hyperium/hyper-util.git" }
 reqwest = { version = "0.11" }
 http-body-util = "0.1.0-rc.3" 
-redis = "*"
\ No newline at end of file
+redis = "*"
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Not sure if that is the best solution, as the HSMs might require a modified OpenSSL (@edgeless-project/infineon might know more about that). In any case, we have a bit of a mixture of OpenSSL / native-tls and RustTLS in the project. Afaik RustTLS does not support the HSMs, so we might need to standardize on OpenSSL. Will need to explore how to best do this, but I guess this works for now.